wye/social/utils.py

```diff
@@ -0,0 +1,22 @@
+def get_message(context):
+    workshop = context.get('workshop', None)
+
+    if workshop:
+        return construct_message(workshop, context)
+    else:
+        # Other messages for non-workshop creation actions
+        return None
+
+
+def construct_message(workshop, context):
+    date = workshop.expected_date
+    topic = workshop.workshop_section
+    organization = workshop.requester
+    workshop_url = context.get('workshop_url', None)
+    message = "{} workshop at {} on {} confirmed! Details at {}".format(
+        topic, organization, date, workshop_url)
+    if len(message) >= 140:
+        message = "{} workshop on {} confirmed! Details at {}".format(
+            topic, date, workshop_url)
+
+    return message
```

Comment:
Ha, being a method on the model sounds good. Will make that change. :+1: 

As for the 140 char limit, I will get to writing the test at some point, but for the time being, the following is the split up of lengths for workshops:
`http://pythonexpress.in/workshop/11/` -> MaxLength = 40
`Web development` -> MaxLength = 20
`2015-10-30` -> MaxLength = 10
'Institution name is so big!" -> MaxLength = 30

So the entire text will read as:

`Web development workshop at institution name is so big! on 2015-10-30 confirmed! Details at http://pythonexpress.in/workshop/11/`

Length this 128, which means we still have a buffer of 12.

The only thing that can go out of hand is the institution name. So that's what the check does. If greater than 140, get the institution name out.
I believe the url_length is fixed as I showed above. I've accounted upto 6 digits at the end of that url.
The topics defined in PythonExpress currently are short. `Web development` is the biggest one I could think of and that has been accounted for here.